src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1131,6 +1165,25 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                 });
             },
 
+            get _structuralVariantCountBy() {
+                const urlSelection =
+                    (vertical
```

Comment:
99% sure this exceeds the scope of this PR, but I don't like all the `vertical ? {} : {}` boilerplate we have. I think a better solution would be two have a type that represents all the axis dependant state, then maintain a Map<Boolean, ThatState>, with two entries, one for vertical == true, and one for vertical == false.

That way statements like this just turn into `this.state.axisState[vertical].plots_selection`.

Reply:
Yeah, this should a separate effort